apps/nextjs-website/src/_contents/send/api.ts

```diff
@@ -5,6 +5,7 @@ import { sendApiPath } from '@/_contents/send/apiPath';
 export const sendApi: ApiData = {
   ...sendApiPath,
   product: send,
+  specURLsName: 'Documentazione API di Send',
```

Comment:
```suggestion
  specURLsName: 'Documentazione API di SEND',
```
SEND should be uppercase